exec/cnex/lib/binary.c

```diff
@@ -0,0 +1,478 @@
+#include "binary.h"
+
+#include <assert.h>
```

Comment:
Do you really need all of these includes? Seems like a lot, but I haven't checked them.

Reply:
Actually, a quick test seems to prove that I don't need any of them.  However, other compilers might feel differently about it?